technical_liaisons.md

```diff
@@ -0,0 +1,25 @@
+# Technical Liaisons
+
+DLS will designate a technical liaison to Each project in its portfolio.
+
+## Why Technical Liaisons
+
+* Ensure that each of our projects are stewarded, looked over, and thought about reliably.
+* Empower developers to learn deeply about the context and stakeholder use cases for a particular application.
+* Act as a point of contact for the product owner to support interactions with stakeholders.
+* Bring knowledge of issues and talking points back to the team for implementation discussion and prioritization.
+* Promote the needs of their application to the team to ensure it isn't lost in the shuffle.
+
+A technical liaison is not necessarily the technical authority or expert on the
+codebase they steward. Technical and architectural decisions are a team effort.
+
+## Responsibilities of a Technical Liaisons
+
+1. Collaborate closely with the product owner to prioritize issues and ensure the backlog is groomed appropriately. This work may also be delegated or shared.
+1. Ensure code review is happening on the project.
```

Comment:
This might be for the retro. But is this code review/documentation step a responsibility of the Liaison? I wonder if it's putting too much pressure on them, and it's an artifact of the "technical lead" role from the past?

Reply:
I left it in for now to give us time to discuss